xsensmt/XsensMT.cpp

```diff
@@ -364,6 +364,18 @@ bool XsensMT::open(yarp::os::Searchable &config)
         return false;
     }
 
+    if(config.check("reset"))
```

Comment:
I suggest adding a more descriptive string like `resetSensorOrientation`. It could be clearer!